src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.Unix.cs

```diff
@@ -52,5 +52,12 @@ public SecureString? Password
             get { throw new PlatformNotSupportedException(SR.Format(SR.ProcessStartSingleFeatureNotSupported, nameof(Password))); }
             set { throw new PlatformNotSupportedException(SR.Format(SR.ProcessStartSingleFeatureNotSupported, nameof(Password))); }
         }
+
+        [SupportedOSPlatform("windows")]
```

Comment:
Are we throwing PNSE now on Unix as part of reserving the possibility of implementing this in the future? Or we think this is permanently Windows only?